Makefile

```diff
@@ -49,3 +49,17 @@ clean: ## Clean generated files
 update-flake: ## Update the Nix flake.lock file, requires Nix installed with flake support, see: https://nixos.wiki/wiki/Flakes
 	nix flake update
 
+.PHONY: build-wayland
+build-wayland: ## Build wayland backend
+	python libqtile/backend/wayland/cffi/build.py
```

Comment:
Dumb question: does this happen automatically when doing something like `uv tool install ...` somehow?